.golangci.example.yml

```diff
@@ -449,6 +449,12 @@ linters-settings:
     servingv1: knative.dev/serving/pkg/apis/serving/v1
     # using `autoscalingv1alpha1` alias for `knative.dev/serving/pkg/apis/autoscaling/v1alpha1` package
     autoscalingv1alpha1: knative.dev/serving/pkg/apis/autoscaling/v1alpha1
+  gomodreplace:
+    # Allow local `replace` directives. Default is false.
+    local: false
+    # List of allowed `replace` directives. Default is empty.
+    allow-list:
+      - launchpad.net/gocheck
```

Comment:
which of these does this allow:
- `replace launchpad.net/gocheck v0.0.0 => /path/to/gocheck`
- `replace launchpad.net/gocheck v0.0.0 => example.com/path/to/gocheck v0.0.0`
- `replace example.com/path/to/gocheck v0.0.0 => launchpad.net/gocheck v0.0.0`

Reply:
currently only the last

Reply:
that seems reasonable, as long as it is clear that `local: true` is not required to get the second

Reply:
I change the behavior https://github.com/ldez/gomodreplace/blob/13fd25eefbe7e332e3d67b7497365363ae58e59f/gomodreplace_test.go#L29